exercises/concept/weighing-machine/.meta/Exemplar.cs

```diff
@@ -1,80 +1,40 @@
 using System;
 
-enum Unit
-{
-    Pounds,
-    Kilograms
-}
-
 class WeighingMachine
 {
-    private const decimal POUNDS_PER_KILOGRAM = 2.20462m;
+    private int _weight;
 
-    private decimal inputWeight;
+    public WeighingMachine(int precision)
+    {
+        Precision = precision;
+    }
 
-    public Unit Unit { get; set; } = Unit.Kilograms;
+    // Goal: getter-only property initialized in constructor
+    public int Precision { get; }
 
-    public decimal InputWeight
+    // Goal: auto-implemented get/set property with initial value
+    public int TareAdjustment { get; set; } = 5;
+
+    // Goal: explicit get/set property
+    public int Weight
     {
         get
         {
-            return inputWeight;
+            return _weight;
         }
         set
         {
-            if (value < 0)
-            {
-                throw new ArgumentOutOfRangeException("weight cannot be negative");
-            }
-
-            inputWeight = value;
-        }
-    }
-
-    public decimal DisplayWeight
-    {
-        get
-        {
-            return ApplyTareAdjustment(inputWeight);
+            if (value < 0) throw new ArgumentOutOfRangeException();
+            _weight = value;
         }
     }
 
-    public USWeight USDisplayWeight
+    // Goal: getter-only property with logic 
+    public int DisplayWeight
```

Comment:
I just noticed that we're not actually using the `Precision` property anywhere. Maybe we change this property to a `string` instead of an `int` and have it use the precision? So its output would look like: "60.5 kg" for precision `1`, and `60.58` for precision`2`, etc..